template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/knox/KnoxServiceConfigProvider.java

```diff
@@ -75,7 +78,16 @@ public boolean isConfigurationNeeded(CmTemplateProcessor cmTemplateProcessor, Te
 
     @Override
     protected List<ApiClusterTemplateConfig> getRoleConfigs(String roleType, TemplatePreparationObject source) {
-        return emptyList();
+        List<ApiClusterTemplateConfig> configList = new ArrayList<>();
+        if (source.getProductDetailsView() != null
+                && isKnoxDatabaseSupported(source.getProductDetailsView().getCm(), getCdhProduct(source), getCdhPatchVersion(source))) {
+            RdsView knoxGatewayRdsView = getRdsView(source);
+            if (knoxGatewayRdsView.isUseSsl()) {
+                configList.add(config(GATEWAY_DATABASE_SSL_ENABLED, "true"));
+                configList.add(config(GATEWAY_DATABASE_SSL_TRUSTSTORE_FILE, knoxGatewayRdsView.getSslCertificateFilePath()));
```

Comment:
How will this work? The `getSslCertificateFilePath()` points to a PEM cert bundle, whereas the Knox property suggests a Java truststore.